com.github.fabiocolacio.marker.json

```diff
@@ -27,7 +27,7 @@
         "sources": [
           {
             "type": "archive",
-            "url": "https://www.abisource.com/downloads/enchant/1.6.0/enchant-1.6.0.tar.gz",
+            "url": "http://www.abisource.com/downloads/enchant/1.6.0/enchant-1.6.0.tar.gz",
```

Comment:
Why the change?

Reply:
they didn't update their certificate, Flathub won't accept it so I figured I might just as well use the non-secure version.

Reply:
It's a 10 year old package... I don't expect them to update it any time soon.

Reply:
there are new issue with letsencrypt certificates I need to fix.

Try 1.6.1 from github:

https://github.com/AbiWord/enchant/releases/tag/enchant-1-6-1

Reply:
Thanks for the update. 1.6.1 seems to work fine so I'll use that version instead. Perhaps also change the info on the website, because I was unaware that a newer version was hosted on a different site.

Reply:
Like this?

<img width="591" alt="image" src="https://user-images.githubusercontent.com/114441/112473270-82ff6400-8d44-11eb-80a0-5a344c81ae4b.png">

Reply:
FWIW the cert problem was fixed. But github being the official stuff, it's no longer relevant.